core/modules/update/update.module

```diff
@@ -193,10 +193,10 @@ function update_menu() {
   }
   // Customize the titles of the action links depending on where they appear.
   // We use += array() to let the translation extractor find these menu titles.
-  $items['admin/reports/updates/install'] += array('title' => 'Install module, theme or layout');
+  $items['admin/reports/updates/install'] += array('title' => 'Install module, theme, or layout template');
   $items['admin/modules/install'] += array('title' => 'Install module');
   $items['admin/appearance/install'] += array('title' => 'Install theme');
-  $items['admin/structure/layouts/install'] += array('title' => 'Install layout');
+  $items['admin/structure/layouts/install'] += array('title' => 'Install layout template');
```

Comment:
Let's hold off on this change for now, as we should update the links for modules & themes at the same time. Perhaps as part of the addition of project browser!  See https://github.com/backdrop/backdrop-issues/issues/1757